src/main/java/io/stargate/sgv2/jsonapi/api/model/command/clause/filter/FilterClause.java

```diff
@@ -21,8 +23,16 @@
              {"name": "Aaron", "country": "US"}
               """)
 public record FilterClause(LogicalExpression logicalExpression) {
-  public void validate(CommandContext commandContext) {
-    IndexingProjector indexingProjector = commandContext.indexingProjector();
+
+  public <T extends SchemaObject> void validate(CommandContext<T> commandContext) {
```

Comment:
I think that in this case `T` is not needed due to use of ".asCollectionContext()".
But not sure if `validate(CommandContext<?>)` is legal -- if not, would need to be

   `validate(CommandContext<? extends SchemaObject>)`

which is as clumsy as defining `T`. If plain `?` is accepted, that'd make sense here tho.

Reply:
Changed to just use `CommandContext<?>`.